samples/samples/snippets.py

```diff
@@ -2428,6 +2511,165 @@ def enable_fine_grained_access(
     # [END spanner_enable_fine_grained_access]
 
 
+# [START spanner_insert_proto_columns_data_with_dml]
+def insert_proto_columns_data_with_dml(instance_id, database_id):
```

Comment:
The samples should be consistent across the languages, Can you please recheck all the samples w.r.t to Go & Java Client?

Reply:
Java has samples only for mutations. However, I think we should have samples that shows both DML and mutations for read and write. Let's discuss offline on this and conclude the samples needed.